### PR TITLE
Add docs to dev-kit

### PR DESCRIPTION
**Pull request by kunicmarko20**
Does this mean that docs config will now be handled by dev-kit? or we should make PR on all repos to fix docs?

### CODE REVIEW

File: project/docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'Sonata ~ AdminBundle'
+copyright = u'2010-2015, Thomas Rabaix'
```
Comment (OskarStark) [inline project/docs/conf.py]: we should change this to `2010-2018`

Comment (greg0ire) [reply project/docs/conf.py]: Or just 2010 IIRC

Comment (kunicmarko20) [reply project/docs/conf.py]: https://github.com/sonata-project/dev-kit/blob/master/project/LICENSE#L3 yep, just 2010, should I fix the style thing in a different PR?

Comment (OskarStark) [reply project/docs/conf.py]: No we can fix it here IMO

**Comment by OskarStark** [comment]
@Soullivaneuh, please merge this

File: project/docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+source_suffix = '.rst'
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'Sonata ~ AdminBundle'
```
Comment (OskarStark) [inline project/docs/conf.py]: Admin Bundle needs to be replaced by a template var imo

File: project/docs/Makefile [hunk 3]
```diff
@@ -0,0 +1,153 @@
+# Makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
```
Comment (ElectricMaxxx) [inline project/docs/Makefile]: @kunicmarko20 can you add the env var usage here as i did in https://github.com/sonata-project/SonataSeoBundle/pull/282 pls doing so we can locally use a separate environment, which is common in python projects

**Comment by kunicmarko20** [comment]
fixed.

**Comment by OskarStark** [comment]
@Soullivaneuh please merge

**Comment by Soullivaneuh** [comment]
I don't understand the goal here. The doc is generated by an another private repository.

**Comment by kunicmarko20** [comment]
The goal is to make dev-kit handle shared files across repositories, in this case requirements.txt because sphinx 1.8.0 is broken and we need to skip it. If dev-kit can’t do this then we need to make PR on all repositories.

**Comment by ElectricMaxxx** [comment]
@Soullivaneuh i really like it also, as it gives the oportunity to integrate a common testing proccess accross the packages for docs. The first think after a merge will be a pull on our cmf fork :-) cause i currently discuss to pull docs into single packages for [cmf also](https://github.com/symfony-cmf/symfony-cmf/issues/263) (discussion seems to become hard, but lets see)

**Comment by kunicmarko20** [comment]
@Soullivaneuh can this work? or can I close it?

File: src/Console/Command/AutoMergeCommand.php [hunk 4]
```diff
@@ -51,7 +51,7 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
     {
         parent::initialize($input, $output);
 
-        $this->projects = count($input->getArgument('projects'))
+        $this->projects = \count($input->getArgument('projects'))
```
Comment (Soullivaneuh) [inline src/Console/Command/AutoMergeCommand.php]: Not really related to the PR.

Comment (kunicmarko20) [reply src/Console/Command/AutoMergeCommand.php]: yes, but the build was failing

Comment (kunicmarko20) [reply src/Console/Command/AutoMergeCommand.php]: should I create another PR just for CS?

**Pull request closed without merging**